File: algorithms/pyVer/sort/BubbleSort.py

arr = [int(input()) for i in range(0, int(input()))]

def bubblesort(arr):

    temp = 0;

    for i in range(0, len(arr)):
        for j in range(0, len(arr) - i - 1):
            if (arr[j] > arr[j+1]):
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
    return arr

arr = bubblesort(arr)

for i in arr:
    print(i)

File: algorithms/javaVer/sort/BubbleSort.java
package sort;

import java.util.Arrays;

public class BubbleSort {

	int[] bubbleSort(int[] intArray) {

		int temp = 0;
		for (int i = 0; i < intArray.length; i++) {
			for (int j = 0; j < intArray.length - i - 1; j++) {
				if (intArray[j] > intArray[j + 1]) {
					temp = intArray[j];
					intArray[j] = intArray[j + 1];
					intArray[j + 1] = temp;
				}
			}
		}
		return intArray;
	}
}

class Main {
	public static void main(String[] args) {

		int[] intArray = {4, 2, 1, 4, 3, 9, 7, 8, 6, 10};

		BubbleSort bubbleSort = new BubbleSort();

		System.out.println(Arrays.toString(bubbleSort.bubbleSort(intArray)));

	}
}